content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -2,40 +2,122 @@
 Cycle counts
 ============
 
-In most companies, the stock is only counted once a year. That's why by default, after making an
-inventory adjustment in Odoo, the scheduled date for the next count is set on the 31st of
-December. However, for some businesses it's crucial to have an accurate inventory count at all
-times.
+For most companies, warehouse stock only needs to be counted once a year. This is why, by default,
+after making an *inventory adjustment* in Odoo, the scheduled date for the next inventory count sets
+for the 31st of December of the current year.
 
-The goal of cycle counts is to keep critical stock levels accurate by counting more often at key
-locations.
+However, for some businesses, it's crucial to have an accurate inventory count at all times. These
+companies use *cycle counts* to keep critical stock levels accurate. Cycle counting is a method by
+which companies count their inventory more often in certain *locations*, to ensure that their
+physical inventory counts match their inventory records.
```

Comment:
physical inventory counts match their inventory records in Odoo.

Not sure if necessary- but maybe just specifying even more that it is the data vs the physical.